src/main/java/net/openhft/chronicle/queue/impl/single/SingleChronicleQueue.java

```diff
@@ -229,7 +229,14 @@ protected CycleCalculator cycleCalculator(ZoneId zoneId) {
     @Deprecated(/* to be removed in x.23 */)
     @NotNull
     StoreTailer acquireTailer() {
-        return ThreadLocalHelper.getTL(tlTailer, this, q -> new StoreTailer(q, q.pool));
+        StoreTailer tl = ThreadLocalHelper.getTL(tlTailer, this, q -> new StoreTailer(q, q.pool));
```

Comment:
I notice a lot of the other methods (including `acquireAppender` and `createTailer`) raise an exception if the queue is closed. Should we do it here for consistency? or is there a use case for getting a tailer on a closed queue?

Reply:
Yes - good spot - we should check for closed

Reply:
I added the check.

Previously:
- if `acquireTailer` was called for the first time on a closed Queue it would throw `ClosedIllegalStateException` due to something downstream of the `StoreTailer` constructor.
- if `acquireTailer` was called for a subsequent time on a closed Queue it would not throw

So the change just means you'll now get that error regardless if its the first or subsequent call to `acquireTailer` on a closed queue.